portal-web/docroot/html/taglib/ui/user_group_search_container_results/page.jsp

```diff
@@ -25,44 +25,24 @@ SearchContainer userGroupSearchContainer = (SearchContainer)request.getAttribute
 if (Validator.isNotNull(searchTerms.getKeywords())) {
 	useIndexer = true;
 }
-
-Indexer<?> indexer = IndexerRegistryUtil.nullSafeGetIndexer(UserGroup.class);
 %>
 
 <liferay-ui:search-container id="<%= userGroupSearchContainer.getId(request, namespace) %>" searchContainer="<%= userGroupSearchContainer %>">
 	<liferay-ui:search-container-results>
-		<c:choose>
```

Comment:
The "useIndexer" attribute was a bit deceptive to begin with.  Even if a search was made with keywords, we still need to honor the params that are passed into the search function.  Doing an index search no matter what caused too many results to be returned.  In the case of this bug, we have keywords but do NOT want to use the index, so it made sense to use the search method, which would automatically determine when it was appropriate to use the indexer.

If we use `search` instead of `searchUserGroups`, the other two checks (`indexer.isIndexerEnabled() && PropsValues.USER_GROUPS_SEARCH_WITH_INDEX`) are automatically performed inside the service class in the search method, so they would be redundant here.